controllers/users.js

```diff
@@ -43,7 +43,7 @@ const verifyUser = async (req, res) => {
     if (!req.userData?.profileURL) {
       return res.boom.serverUnavailable("ProfileURL is Missing");
     }
-    await userQuery.addOrUpdate({ profileStatus: "PENDING" }, userId);
+    await userQuery.addOrUpdate({ profileStatus: "PENDING" }, userId, req.query.dev);
```

Comment:
Please create a separate variable for req.query.dev and pass it here

Reply:
resolved